zhaquirks/tuya/__init__.py

```diff
@@ -831,6 +831,18 @@ def update_attribute(self, attr_name: str, value: Any) -> None:
         return self._update_attribute(attr.id, value)
 
 
+class TuyaNoBindPowerConfigurationCluster(PowerConfiguration, CustomCluster):
+    """PowerConfiguration cluster that prevents setting up binding/attribute reports in order to stop battery drain."""
+
+    async def bind(self):
+        """Prevent bind."""
+        return (foundation.Status.SUCCESS,)
+
+    async def _configure_reporting(self, *args, **kwargs):  # pylint: disable=W0221
+        """Prevent remote configure reporting."""
+        return (foundation.ConfigureReportingResponse.deserialize(b"\x00")[0],)
```

Comment:
Just guessing, would it be the same that `ConfigureReportingResponseRecord(status=Status.SUCCESS).serialize()`?`
https://github.com/zigpy/zigpy/blob/1f81b162f84328eb96db18add3104d889fa6c923/zigpy/zcl/foundation.py#L438

Reply:
Yep, probably. I think we should also change it in `LocalDataCluster` then. Didn't want to do that in this PR though.

I think I'd leave it like this and then change both (and some other similar stuff) in a future PR(?)

Reply:
Yes, for sure. 👍🏻 (just doing a quick issues review, not trying to criticize nobodies work)